### PR TITLE
Add WiCS-UWARC movie night event post

### PR DESCRIPTION
**Pull request by meltedesco**

### CODE REVIEW

File: content/W2015/W15-event-uwarcmovienight.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: WiCS and UWARC Movie Night
+Date: 2015-02-12 19:00
+Category: Events
+Tags: social
+Slug: uwarc-movie-night
+Author: Melissa Tedesco
+Summary: Join us for a post-midterm movie night at POETs.
```
Comment (ehashman) [inline content/W2015/W15-event-uwarcmovienight.md]: POETS is with a capital S: http://engsoc.uwaterloo.ca/services/poets

If you can fix these three typos that would be great!

**Comment by meltedesco** [comment]
Done! Thanks for letting me know.

**Comment by ehashman** [comment]
Great!

**Comment by ehashman** [comment]
So deploying this actually broke the site due to a typo in category name ("Event" not "Events")---I fixed it :)

**Pull request merged**